### ISSUE TITLE
Gear animations

### ISSUE THREAD

**Issue by Silver-Snail** [racerretrocoder/Flightgear-F-22A-Raptor#44]
Having not worked on the rear gear I cannot say exactly how to solve it, but for the front gear, put the tire (and it's rotating axis) as a subset of Nose_Gear_Cylinder02. Here's what the hierarchy looks like in blender:
![image](https://github.com/user-attachments/assets/df5fba34-2ed1-4b2a-b3f3-dd1d808c623f)
I do have the changed ac file with just that change if you want me to put it on mine, or if you wish you can change it yourself.

**Comment by racerretrocoder** [comment]
I have an idea. What if i made the gear some how stop its animation when the gear was abbout to retract. That might beable to fix it.
I also cannot assign an animation to the axis oniect with out it not being able to be used

**Comment by Silver-Snail** [comment]
> I have an idea. What if i made the gear some how stop its animation when the gear was abbout to retract. That might beable to fix it. I also cannot assign an animation to the axis oniect with out it not being able to be used

Being you used the spin function, unless you "unspin" it (basically setting it back to it's original position) then it's still gonna be spun around the axis which is not in the center of the tire anymore.

**Comment by racerretrocoder** [comment]
have 2 sets of wheels. clipped into each other.
then when the gear is down have the spinning wheels show. then hide the copy wheels
as soon as the gear starts to go up
hide the spinning wheels and show the static wheels.

**Comment by racerretrocoder** [comment]
or i could try to make a nasal script to control the factor and Degrees for the rotation animation instead of using spin

this may seems hard

**Comment by Silver-Snail** [comment]
You could do either of those, but wouldn't that be harder than just putting it as a subset of the gear struts?

**Comment by racerretrocoder** [comment]
Whats a subset? 
Is that a thing with ac3d?

**Comment by Silver-Snail** [comment]
> Whats a subset? Is that a thing with ac3d?

I don't know if AC3D does or not, but it should because it is a thing in the ac format. Does it have a list of the objects?

**Comment by racerretrocoder** [comment]
I think. 
I can press ctrl a and itll tell me all the objects i have selected

**Comment by racerretrocoder** [comment]
Oh Do you mean by object groups instead of just plain objects

**Comment by Silver-Snail** [comment]
> Oh Do you mean by object groups instead of just plain objects

Correct. Put the nwheel2 object inside the Nose_Gear_Cylinder02 group.

**Comment by racerretrocoder** [comment]
Ill work on this next

**Comment by racerretrocoder** [comment]
idk if i can fix the gear, what ever i try it just makes it worse 😆

**Comment by racerretrocoder** [comment]
They now spin again. But they wont spin just after when the gear goes up. Idk how to fix them from doing that. I tried adjusting animation placement (like movement first then spinanimation, vice versa) but couldnt figure it out

**Comment by racerretrocoder** [comment]
Ment to say that they would spin... But not in the correct way what so ever
